src/controls/tab-bar/focus.js

```diff
@@ -81,6 +88,7 @@ function getAdjacentControl(control, { previous = false } = {}) {
 
 // Recursively searches for the closest parent tab list
 function getClosestTabList(el) {
+  /* istanbul ignore next */
```

Comment:
This isn't possible to test without changing the markup of the component. If this situation does occur, we'll get an error on `undefined.querySelectorAll('[role="tab"]'))`.

However, it still feels good to me to have this guard in a recursive function? Let me know if you think we should remove it outright